packages/hash/api/src/graphql/typeDefs/ontology/entity-type.typedef.ts

```diff
@@ -0,0 +1,56 @@
+import { gql } from "apollo-server-express";
+
+export const entityTypeTypedef = gql`
+  scalar EntityType
+
+  type PersistedEntityType {
+    entityTypeVersionedUri: String!
+    createdBy: ID!
+    schema: EntityType!
+    # TODO: we might need something like
+    # "referencedPropertyTypes: [PersistedPropertyType!]"
+  }
```

Comment:
Nit: would be nice to get documentation back for these fields, as I don't expect this to be a temporary GQL type

Reply:
Added in ea4f0557a9690aa30f14399a2f4ac6000f0ec15b for all ontology type `Persisted*` types